actions/test/action.yml

```diff
@@ -0,0 +1,175 @@
+name: "Test"
+description: "Action to test Node.js projects with support for coverage reporting and pull request annotations"
+author: hoverkraft
+branding:
+  icon: check-square
+  color: blue
+
+inputs:
+  working-directory:
+    description: |
+      Working directory where test commands are executed.
+      Can be absolute or relative to the repository root.
+    required: false
+    default: "."
+  container:
+    description: "Whether running in container mode (skips checkout and node setup)"
+    required: false
+    default: "false"
+  coverage:
+    description: |
+      Code coverage reporter to use. Supported values:
+      - "codecov": Upload coverage to Codecov
+      - "github": Use ReportGenerator for PR comments with coverage reports
+      - "": No coverage reporting
+    required: false
+    default: ""
+  coverage-files:
+    description: |
+      Path to coverage files for reporting.
+      Supports multiple formats (Cobertura, OpenCover, lcov, etc.).
+      Can be a single file or multiple files separated by semicolons.
+    required: false
+    default: "coverage/cobertura-coverage.xml"
```

Comment:
coverage files should be able to be autodetected also (from the lint command, and / or common widly used file name / path)